integration-tests/src/test/java/oracle/kubernetes/operator/ITOperator.java

```diff
@@ -622,39 +616,55 @@ public void testCustomSitConfigOverrides() throws Exception {
     }
     Domain domain12 = null;
     boolean testCompletedSuccessfully = false;
-    String createDomainScriptDir =
-        BaseTest.getProjectRoot() + "/integration-tests/src/test/resources/domain-home-on-pv";
     try {
 
-      // cp py
-      Files.copy(
-          new File(createDomainScriptDir + "/create-domain.py").toPath(),
-          new File(createDomainScriptDir + "/create-domain.py.bak").toPath(),
-          StandardCopyOption.REPLACE_EXISTING);
-      Files.copy(
-          new File(createDomainScriptDir + "/create-domain-custom-sit-config.py").toPath(),
-          new File(createDomainScriptDir + "/create-domain.py").toPath(),
-          StandardCopyOption.REPLACE_EXISTING);
+      this.overrideCreateDomainScript(domain12DomainScript);
 
       domain12 = testDomainCreation(domain12YamlFile);
       domain12.verifyDomainCreated();
       testBasicUseCases(domain12);
       testAdminT3ChannelWithJMS(domain12);
-      // testAdvancedUseCasesForADomain(operator1, domain11);
+      // testAdvancedUseCasesForADomain(operator1, domain12);
       testCompletedSuccessfully = true;
 
     } finally {
-      Files.copy(
-          new File(createDomainScriptDir + "/create-domain.py.bak").toPath(),
-          new File(createDomainScriptDir + "/create-domain.py").toPath(),
-          StandardCopyOption.REPLACE_EXISTING);
+      this.unoverrideCreateDomainScript();
       if (domain12 != null && (JENKINS || testCompletedSuccessfully)) {
         domain12.destroy();
       }
     }
     logger.info("SUCCESS - testCustomSitConfigOverrides");
   }
 
+  @Test
+  public void testIntrospectorSitConfigOverrides() throws Exception {
```

Comment:
Add javadoc with description of the test.

Reply:
Test is now removed